public/contexts/index.tsx

```diff
@@ -42,7 +56,10 @@ export const useSearchRelevanceContext = () => {
 };
 
 export const SearchRelevanceContextProvider = ({ children }: { children: React.ReactNode }) => {
+
   const [documentsIndexes, setDocumentsIndexes] = useState<DocumentsIndex[]>([]);
+  const [documentsIndexes1, setDocumentsIndexes1] = useState<DocumentsIndex[]>([]);
+  const [documentsIndexes2, setDocumentsIndexes2] = useState<DocumentsIndex[]>([]);
```

Comment:
not very related to this PR: i have no experience with search relevance plugin and i see many variables are in the form of `var`, `var1`, `var2`. what's the difference?